### DISCUSSION TITLE
Thoughts on the Scaling Dilemma for SPs & Potential Solutions

### DISCUSSION THREAD

**Discussion by adam-dltx** [filecoin-project/FIPs#583] [Enhancements - Misc]
## Motivation

@Stefaan-V’s post highlighted the main challenges of the Filecoin network. This discussion topic aims to advance the conversation on a major challenge that many SPs face: liquidity required to scale as a Storage Provider (SP).

Due to pledge requirements, as Filecoin Storage Providers scale and bring on new enterprise clients, they must have significant access to liquid FIL. The Initial Pledge requirements grow in proportion to the size of the client; therefore, the larger the enterprise client, the more access to FIL is required. Requiring large amounts of Initial Pledges means that liquidity will be a major driver of network growth. Without it, SPs will be unable to fulfill the primary function they serve. 

The incentives introduced by FIL+ are working, and SPs are building sales pipelines of useful data, but SPs are handicapped by their access to FIL. With the increased incentives, FIL+ increases the necessary Initial Pledge. Unfortunately, this can lead to the need for massive collateral which prevents Storage Providers from onboarding these large data sets even if the offline deal is closed. Thus, access to FIL, not deal flow, hardware, or operations, becomes the primary challenge for SPs. 

### To Borrow or Buy
A lack of FIL already on the balance sheet leaves these SPs with two options: borrow or buy.

When markets were far more liquid, SPs could borrow FIL to fill their Initial Pledge needs. As liquidity dried up, Storage Providers competed for large FIL loans, increasing interest rates. This crunch became the major blocker for some SPs when it came to adding additional value to the network.

As large loans of FIL can be hard to come by, it can be nearly impossible to close a deal and secure a loan. Compound this problem with the extra costs of taking a loan too early or at the wrong interest rate, and Storage Providers see already tight margins eliminated. Loans aren’t a perfect solution, but they do keep the machine running when they are available. 

Purchasing FIL is not an option for some SPs as adding price risk and hedging is outside their core business. Even if it was an option, it’s almost impossible to raise capital to purchase FIL, as the broader crypto market is locked out of the capital markets. @DSS-AL raises this point in the Revisiting a Sector Duration Multiplier discussion. Even if it was possible, the amount of money required for successful SPs would be tens of millions of dollars annually on top of the capital, hardware, direct costs, and overhead. 

Many SPs, including DLTx, want to avoid direct price exposure to the token at the pledge level as it introduces too much risk for a storage business. If price risk management is forced upon SPs, customers will feel the brunt of the additional costs stemming from that activity. It is both an unnecessary distraction and costs for all players. 

### Visualizing the Issue

Growing your SP business with FIL+ becomes a significant challenge at large scales. The chart below demonstrates the problem:

| PiB  | Total Collateral  | FIL/TiB (FIL+)  | FIL Price  | $ Amount  |
| :------------: | :------------: | :------------: | :------------: | :------------: |
 | 	1	 | 	65,555	 | 	64.019	 | 	$3.00 | $196,666.37	 | 
 | 	10	 | 	655,555	 | 	64.019	 | 	$3.00	 | 	$1,966,663.68 | 
 | 	100	 | 	6,555,546	 | 	64.019	 | 	$3.00	 | $19,666,636.80 | 
 | 	250	 | 	16,388,864	 | 	64.019	 | 	$3.00	 | 	$49,166,592.00 | 

When charted, it looks like this:

<img width="745" alt="image" src="https://user-images.githubusercontent.com/73503641/210113798-5e6050bb-30a8-4b18-87f9-25f37c5a3860.png">

#### The Liquidity Dilemma 

SPs should be able to operate without the necessity of purchasing the token outright. Companies with a conservative general operating thesis may be prevented from buying the FIL outright. For these companies, they should be able to earn their way into a FIL position. A business’ fortunes should not solely depend on the price of FIL on its balance sheet, rather it should be determined by how successful we are at being a storage provider. With present conditions, it’s plain difficult to build a successful Storage Provider business from the ground up. 

SPs attempting to operate with tens and hundreds of PiB clients, run into a liquidity crunch nearly every time they close a deal. This leads to spending more time looking for and negotiating FIL loans than working the sales pipeline. To their detriment, their ability to borrow FIL determines our success. Matters are only worse with recent market conditions where it’s been difficult, if not impossible, to find lenders at any interest rate. 

The Filecoin token economics leaves some SPs in an uncomfortable situation where we have demand for services that are ready to be rendered (infrastructure in place), without the ability to onboard the customer(s). 

## Ideas for Feedback from the Community

A preface: Despite the DLTx’s long history with Filecoin, I’m a relatively new entrant to the ecosystem. The thoughts below are relatively fresh ideas and may have some incorrect assumptions. Additionally, I tried to be thoughtful to prevent the rehashing of prior conversations, but it’s difficult to avoid this. As you might imagine, I’m missing quite a bit of history here.

### Near Term: Manually Coordinate for SPs

In the near term, token holders need to lend FIL to SPs actively growing the network. Idle FIL should be put to use and given to those that will use it for the benefit of the network. Protocol Labs and/or Filecoin Foundation should help facilitate the matchmaking required to get deals done. While it does not solve the macro issue, it is necessary to buy SPs time during this lending crunch. 

### Mid-Term: Economics Adjustments

Economics should be adjusted to enable SPs to grow while balancing the protection of clients. Tokenomics should not prevent the growth of the network - it should enable the network to grow sustainably. Below are some hot off-the-press ideas that are fodder for discussion (some will certainly be unpopular). 

#### Vested Earnings

Without needing to change the underlying pledge commitments, we leverage the current rewards system to redirect vesting rewards as pledges, effectively recommitting otherwise-to-be liquid tokens back into the system. 

Filecoin rewards are distributed 25% upfront and 75% of the time after the fact over 180 days. There may be a way to leverage this to our advantage. We imagine a system where SPs can commit the 75% vesting tokens to an initial pledge on another deal. In this way, SPs are incentivized to bring on new sectors immediately with tokens they would otherwise receive over a long period. 

This redirection of tokens provides value in multiple ways: 
1. It would demonstrate an SPs commitment to Filecoin (they are doubling down)
2. This puts tokens to work that are otherwise idle. In our case, we would lock these tokens up for the maximum amount of time possible to assist with our long-term deal flow.
3. It would lower the amount of overall liquid tokens available to sell into the market 

This is not a new idea, @hyunmoon suggested this [very idea](https://github.com/filecoin-project/FIPs/issues/14#issuecomment-708253164 "very idea") in 2020. 

> Would it be possible to allow the rest to be used as pledge collateral?
25% is better than 0% but I'm afraid it's still not enough to keep our machines running.
We need liquidity for pledge collateral more than for withdrawal.

If you continue to read that thread, you’ll notice that the SPs faced similar challenges in that market. @momack2 [goes on to say](https://github.com/filecoin-project/FIPs/issues/14#issuecomment-711091308 "goes on to say") that the current setup was a simple solution that can be implemented without much risk. Importantly, momack2 goes on to say that we should look into this very solution (or something like it). It is unclear if this was ever followed up on. 

#### Zero-Upfront Pledge

Another interesting scenario would be to use future earnings to completely cover the Initial Pledge. In such a scenario, all earnings would be deferred as collateral until the Initial Pledge was entirely covered. For the avoidance of doubt, the SP would have zero upfront Initial Pledge but also would not receive a single FIL as a reward until the Initial Pledge is in place. 

In the following example of a verified deal, the SP defers ~480 days of earnings to seal the sector in the hopes of earning in the long term. It assumes that rewards are immediately available for use as collateral. 

<img width="759" alt="image" src="https://user-images.githubusercontent.com/73503641/210113821-115fe73d-3fb6-474b-b299-3b3eadd762e7.png">

A mechanism like this would enable those without large amounts of FIL to quickly seal new sectors and expand as business came in as they would not need to have liquid FIL to pledge. They could immediately bring on new sectors when demand is present without needing to negotiate with lenders or sellers for additional tokens. 

It also may be worth exploring a variable Initial Pledge, where miners can pledge some, but not all of an Initial Pledge, and earn into the rest as demonstrated above. As an example, a SP would put down 50% of the Initial Pledge and have to use future rewards to make up the rest. This could be helpful for deals where clients are willing to pre-pay for services offline and then SPs can use that to purchase part of the Initial Pledge. Alternatively, if an SP had FIL on hand, but not enough for the entire pledge, they should be able to move forward with a verified deal. 

The obvious trade-off is cash flow, but that would need to be weighed against the upside of onboarding new sectors whenever required. Allowing this Initial Pledge bootstrapping mechanism would enable those that are heavily dependent upon loans to reduce their overall costs, as they could optimize their FIL debt vs long-term earnings by mixing and matching the options. This could be like a portfolio that was 50% loan-backed deals that were cash-flowed and 50% deals that were long-term FIL accrual plays. The loan-backed deals would pay today and the other deals would begin to cash flow in ~16 months. 

There are a few potential hangups. This scenario could only be leveraged with long-term deals and there is very little protection for clients in the early days of the storage. This may work for some clients in certain situations, but possibly not for every client, especially those with critical data. There is also a question of how this interacts with FIP-0004 and its token vesting.

A model like this could likely only be used for verified deals, as it could be an attack vector if left unchecked. SP could accumulate large amounts of mining power without owning any of the tokens upfront. It may have implications for tokenomics as well. If anyone can earn into a position, it may have adverse consequences on the underlying token flows. More thought and research may need to be done on this topic. 

#### Scale Multiplier

To allow for the scale that is required today, an alternative method would be to scale down Initial Pledge requirements as SP sizes increase. The logic behind this idea is that as an SP scales, they are less likely to be a bad actor. These are the providers that are disproportionately affected by the large FIL requirements that are unable to be met in the current environment. 

Granting those SPs with some flexibility on FIL+ deals to onboard clients with less Initial Pledge would grease the skids and enable those providers to continue to scale. 

An example of what this might look like is listed below:

<img width="758" alt="image" src="https://user-images.githubusercontent.com/73503641/210113836-d7e69ab3-e1ae-4ad9-97de-6fb6eec3941d.png">

When compared to the chart above, it is immediately apparent that the reduction in capital requirements is dramatic. So much so that we believe this would move the needle for most SPs. This small change opens the door to more useful storage on the network by eliminating the overwhelming need for FIL as operators scale. 

The open questions (not exhaustive) would be: 

1. Does this change hurt clients or the security of the network in any way?
2. Does this have the potential to disproportionately affect small SPs as they would be forced to pay higher rates, even if the dollar amounts are much smaller? 
3. Would the scale multiplier be marginal or retroactive?

#### Trust Score

Rather than simply using the same Initial Pledge Rate for every party, we could leverage on-chain statistics or the social layer already in place with FIL+ to institute a Trust Score that could be used to lower Initial Pledge requirements (as a discount on the prevailing rate). The higher the score, the lower your Initial Pledge requirements. 

The factors required to determine a Trust Score would be determined in the future, but could be related to on-chain fault statistics or could be determined by contributions to the network (in the form of useful storage). The basic premise is that those playing by the rules, and contributing to the network in a real way should be rewarded and assisted in furthering the network. 

A simple curve could be used to determine the collateral rating. Note: in this example, the lowest Trust Score pays a premium on Initial Pledge and the highest score (10) receives the highest discount on the prevailing rate.

<img width="771" alt="image" src="https://user-images.githubusercontent.com/73503641/210113764-2e3d5cf3-a82f-4961-9e3c-191f6e862a1f.png">

This problem eliminates the large SP bias in the prior idea but is more likely to be gamed if we are solely relying on social coordination. 

Open questions:
1. How are changes to scores managed (would it be retroactive?)? 
2. If a credit rating is wrong, who’s at fault (would we hold the social layer accountable?)?
3. What factors determine a score?
4. Is there a size requirement (can an SP run the minimum size setup to max their score and then quickly scale to take advantage of this?)?

Needless to say, this may introduce more problems than solutions.

## Closing Thoughts

Regardless of the solution, the Filecoin ecosystem needs to find an answer to scaling. SPs left to their own devices will struggle to locate FIL required to add useful data to the ecosystem. We should be prepared to adapt the existing economics to work for the system. 

For those of you who are interested in the charts, you can find those [here](https://docs.google.com/spreadsheets/d/151wlyAfdHZO68MaOzOjBO42MxpA_agjip5jwT6OJKDI/edit?usp=sharing). Please make your copy if you wish to play with them.

Edit: For clarity, I changed all references of Service Provider to Storage Provider. Also changed references of miner to Storage Provider.

**Comment by DavidAJohnston** [comment]
Adam,

Thanks for the thoughtful proposals regarding solving the the challenges the Filecoin community and specifically Storage Providers are facing right now.

Out of the four options you explore above I'd rank the Vested Earnings proposal the highest. This leverages an existing mechanism and as you point out aligns the Storage Provider with the community as they are doubling down on their belief in Filecoin. 

The other three proposals may create too much complexity on the technical side, open new risks or change the token economics to advantage certain players over others when it comes to scale.

I'd love to see a deep dive and more specific proposal just on the Vested Earnings solution and how that would be coded / implemented.  

I believe the Filecoin Network is at an important inflection point. Having a solution that allowed Storage Providers to accelerate real world data moving onto the Filecoin Network is THE key to its success going forward.

Best Regards,

David A. Johnston

**Comment by DavidAJohnston** [comment]
Adam,

The other proposal that might be worth exploring in addition to the Vested Earnings one is a Fractionalized FIL Pledge (FFP) arrangement. 

This is where a fraction of the FIL Pledge is committed every month, instead of all up front. So if for example if 1,800 of FIL is be required to be Pledged for 18 months, a Storage Provider commits 100 FIL per month every month during the 18 month period. 

From a token economic perspective the same amount of FIL is pledged and thus the protocol maintains the same amount of demand for FIL in the market and the same amount of collateral securing the sector, but the cost is spread out over time.

This cadence matches better with the real world where a user pays monthly for data storage to their cloud provider. The Storage Provider can then take the payment from the data storage user and buy FIL every month and commit it as Pledge every month. This seems the ultimate organic way to scale usage of Filecoin with the growth of user data storage demand.

Best Regards,

David A. Johnston

**Reply by dd45e640b42e6da7da96faee3996ef7c** [reply]
> This is where a fraction of the FIL Pledge is committed every month, instead of all up front. So if for example if 1,800 of FIL is be required to be Pledged for 18 months, a Storage Provider commits 100 FIL per month every month during the 18 month period.

then the storage power from those pledges needs to be adjusted accordingly! 1/18th in month one, 2/18th in month two and so on. or the sectors cannot be terminated - thats another option. otherwise this has the potential to destroy the chain...

i rather prefer dynamic staking in the boundaries of certain parameters per sector.  but we cannot start giving out storage power for "free"! 

storage is a gimmick - the consensus is the feature of the chain that needs to be protected here. 

if people cannot afford FIL+ x10 multipliers, or soon x5 duration multipliers - then do not use them. store data at x1 and pay the pledge for x1. simple.

**Comment by DavidAJohnston** [comment]
F80ptrk,

I think you make a good point that the Storage Power should be adjusted accordingly to the amount of FIL Pledged.

So if someone starts out with 1 / 18th of the FIL pledge then they have a proportionate amount of Adjusted power. 

I also agree that the consensus must be protected against any bad actors.

Though I believe strongly real data storage must be the driving purpose of Filecoin. In the longterm that is what will drive the value of the network, people using it for its intended purpose. Just as other Networks such as Bitcoin and Ethereum went through their boot strapping phase to build scale and then moved into a focus on transactions, so will Filecoin make the same transition.

Best Regards,

David A. Johnston

**Reply by dd45e640b42e6da7da96faee3996ef7c** [reply]
i recommend everyone sits down with a pen and a paper and calculate how much actual storage space you'd need for a 51% attack on the network with the existing multipliers and the coming ones. 

there is little room left. all that stands in the way is pledge in the end. and yes you can easily get 3PiB/day sealing power rented these days.

**Comment by anorth** [comment]
@nicola and I have thought through similar proposals a while ago too. We wrote up a draft for [network provided loans](https://pl-strflt.notion.site/Network-provided-loans-for-miners-9842a6df780741ff97103a9387f282e1). This encapsulates both the ideas of vested earnings and reduced up-front pledge outlined in the OP, and includes a protocol revenue stream.
- Under-pledging is cast as a _loan from the network_ to the SP to satisfy their pledge debt, collateralised by vesting and future earnings.
- This loan pays interest to the network (by burning it)
- The interest rate is set by network-wide demand for such loans, and an individual SP's amount of under-collateralisation

While I'd be excited to pursue this in more detail, proposals like this have a few challenges which need careful consideration:
- Using vesting/future tokens is not as good for circulating supply as using unlocked tokens. These proposals reduce total locked FIL _as compared with the alternative of using unlocked tokens_ leading to higher supply and pledge requirements.
- These proposals might undermine active lending programs, reduce lending rates (indeed, are designed to), reducing demand to hold tokens, leading to sell pressure
- Both the above might reduce the dollar value of SPs future reward streams, reducing profitability

These tradeoffs highlight the importance of clearly identifying which problems are to be solved.
- If the problem is low returns on storage investment, we don't want to decrease future token values, but instead want to stabilise supply
- If returns are high but the problem is access to capital to invest, then ideas like the above make more sense.

The network may well oscillate between which "problem" is more limiting over time, and we don't want to make slow-moving protocol changes that push too hard one way or the other. With careful design we can probably find parameters (like high network interest rates) or feedback mechanisms (vary with current long-term realised growth rate or token supply growth rate) so that only the "right" mechanism exerts much effect when needed.

**Reply by dd45e640b42e6da7da96faee3996ef7c** [reply]
the problem is a failed business model - they can just stop do fil+ and they have the fil to store the clients data.

**Reply by ashwanth-eth** [reply]
> These tradeoffs highlight the importance of clearly identifying which problems are to be solved.If the problem is low returns on storage investment, we don't want to decrease future token values, but instead want to stabilise supply. If returns are high but the problem is access to capital to invest, then ideas like the above make more sense.

I would pose that the major problem right now is the latter and this will continue to persist moving forward. Circulating supply is high, no-doubt, but the amount available to lend or provide to SPs, seems to only be a small piece of the large circulating supply pie. Breaking this into two parts, I think that: 

- It seems that FIL locked up in 'retail wallets' that may not ever contribute to the ecosystem (individuals who purchased FIL during various flurries of 'ecosystem-hype' and see no need to do anything but hodl) 
- Additionally, @adam-dltx correctly points out that traditional storage business models do not account for the high volatility that cryptocurrency can introduce. Purchasing FIL on the open market market poses a risk to SPs who are used to a 'picks-and-shovels' business model (i.e. many SPs come from the web2 world)

Heavy +1 on pursuing potential solutions moving forward @anorth, while in-parallel enabling FVM DeFi solutions (which might make it easier for FIL holders to lend FIL, thus increasing the amount of FIL that is practically available to lend)

**Comment by steven004** [comment]
A decentralized lending market over FVM might help on this problem if it can: 
- provides a simple way for hodlers to deposit FIL into the system to earn interest
- takes unlocked reward and collaterals and future income from SPs as collateral
- has a dynamical interest mechanism to reflect the relationship between supply and demand

And I believe a decentralized lending market will significantly lower the interest than the current one in the market since individuals could join easily, thus the suppliers will be much more than we have right now. 

We already see a few projects in FVM Space Warp to support leading market, hopefully we do need to wait too long.

**Reply by dmc314** [reply]
I agree @steven004. I think a lot of the ideas mentioned in this thread could be successfully implemented as smart contracts on FVM, add market-based optionality and flexibility, and avoid additional complexity in the core protocol.

**Reply by dmc314** [reply]
For reference, here is a list of the subset of FVM early-builders that have registered their projects, if folks want to follow: https://ecosystem.filecoin.io/?filters=enabled&tags=FVM

It is not complete, but is probably the best public list.

**Reply by dmc314** [reply]
Here is another rawer/WIP list of FVM projects that SPs can watch for: https://airtable.com/shrbUcP3CxFQ8PwvN/tblBMfcmXTOvkXivH

This is filtering today by SP-related use cases:
![image](https://user-images.githubusercontent.com/93460027/211081809-4a00e8da-e030-4a81-bc68-dfe8e612cc7e.png)

**Reply by dmc314** [reply]
(hat tip to @hammertoe and @truckerfling respectively for those links - thanks you two!)

**Reply by momack2** [reply]
You can actually sign up as an SP for glif's upcoming FIL lending pool already: https://docs.google.com/forms/d/e/1FAIpQLSfrDffDpFBLOnwSq5B-NqGYSJ3W6U7q_yrKoHvUHL8QBL0-Og/viewform

![CleanShot 2023-01-19 at 15 43 51](https://user-images.githubusercontent.com/618519/213487226-2497e389-cfbc-4af1-95d0-23be850d43a0.png)

**Comment by adam-dltx** [comment]
First off, I am encouraged by the engagement of this post. Thanks for the feedback. 

## Thoughts on the Comments

I agree that this is the primary problem: 

> * If returns are high but the problem is access to capital to invest, then ideas like the above make more sense.

As an SP, the terms of a deal make sense, but we cannot execute deals due to access to capital to invest in the Initial Pledges. 

Say what you will about FIL+, but there should be an incentive beyond the base rewards rate to perform the effort of bringing useful data to the network. Should these incentives go away over time? Yes, but for now when the network is trying to incentivize network participants to invest in the long-term vision of data storage, it's dangerous to call for the gutting of this program. In a scenario where I can earn the same with or without doing the difficult work of sales, customer support, etc., I'm always going to opt for committed capacity over deals. 

## Timeline Considerations

We need to consider the timeline as part of this discussion. Without that, we may not build the correct tooling to get the correct jobs done. Further, those jobs may vary over time.

- **Near-term** (days/weeks) - I am sure we aren't alone in saying that if we had access to FIL we could close deals and add useful data to the network. Unfortunately, nothing I suggested solves this. Unfortunately, I believe this will be a manual process of FIL owners and SPs getting together to provide the right risk-return ratio. Having a lender of last resort may solve this near-term problem. 
- **Mid-term** (months) - As @steven004 brings up, this is where FVM is well positioned to play a role. A decentralized lending application could solve some of the problems. The questions remain: who has enough FIL to take this risk? If they aren't already participating in lending, why does another lending option solve this (I understand not trusting centralized lenders - that's a real concern)? 
- **Long-term** (quarters) - Economics R&D or changes suggested by myself or others on this thread. @anorth brings up excellent considerations, and we should heed his advice about moving slowly and right-sizing our solutions. @f8-ptrk also brings up good points about protecting consensus. Whatever the solution, we can't throw out the baby with the bathwater. 

## Closing thoughts

The options presented are fodder for discussion. No matter the solution, we're after the long-term success of Filecoin and SPs. As a dedicated storage protocol, it's important not to lose sight of the larger goal of decentralized (useful) data storage. Enabling the good actors within the ecosystem should be top of mind when considering changes, managing risks, and tokenomics in the process.

**Comment by dmc314** [comment]
One note to bear in mind is that the initial pledge per PiB is not fixed, and all other things held equal, it will be coming down soon. This is because it is divided by the max of either the network QAP or the baseline function. QAP will grow faster if a sector duration multiplier is applied, and if that doesn't bring it down faster, the exponential growth of the baseline target function is projected to catch up with it at some point and divide by a larger and larger number.

See here from the Filecoin spec: https://spec.filecoin.io/systems/filecoin_mining/miner_collaterals/#section-systems.filecoin_mining.miner_collaterals.initial-pledge-collateral

><img width="830" alt="image" src="https://user-images.githubusercontent.com/93460027/210884732-e67738cd-aa33-4af1-a591-b4f74940333f.png">

See some projections of the declining pledge (and resultantly stabilizing ROI) in an analysis here: https://github.com/filecoin-project/FIPs/discussions/554#discussioncomment-4605153

**Reply by adam-dltx** [reply]
This is a point worth keeping in mind. Thanks for the clarity @dmc314. I suspect that SPs would have similar issues even considering the numbers from the modeling done here: https://github.com/filecoin-project/FIPs/discussions/554#discussioncomment-4605153. 

I'm hopeful that a combination of this naturally decreasing pledge, better more liquid lending markets with FVM, and some thoughtful economics changes will be the unlock that SPs need to launch and grow their businesses.

**Reply by DSS-AL** [reply]
Thanks for highlighting these simulations @dmc314, however even if the pledge rate halved in the short term, with a FIL price @ $10 for example, we would still need to find a minimum of $32,500,000 in capital over the next 12 months to fulfil anticipated customer demand or as much as $160,000,000 with a 5 year multiplier.

We're just 1 of the many active SPs onboarding customers and storing FIL+ data, there is simply not enough capital (Dollars of FIL) in our ecosystem to support this for all active SPs.

The multiplier will be adopted by SPs where possible in order to survive, this will result in SPs exhausting FIL resources to maximise earnings in the short term. This will result in SPs having to cease onboarding new data as there will not be sufficient capital available to SP on commercially viable terms. This issue is discussed in detail by @adam-dltx in discussion https://github.com/filecoin-project/FIPs/discussions/583

For example, rather than onboard 100 PiB for 1 year, SPs will (where possible) opt to store 20 PiB for 5 years.
